catalyst-gateway/bin/src/rbac_cache/mod.rs

```diff
@@ -0,0 +1,307 @@
+//! A cache for RBAC registrations.
+
+// TODO: Remove the attribute.
```

Comment:
FYI, add your git handle or initials to TODO's so if they get left behind, someone else knows who to ask about them.
If they are an unfinished feature out of scope, link to a github ticket for that if one exists.

Reply:
Good idea, updated.